server/controllers/auth.rb

```diff
@@ -15,13 +15,17 @@ class AuthController < ApplicationController
         .to_json(root: true, :only => [:username, :id])
     else
       requires_role! :admin
-      User.to_json(root: true)
+      User.to_json(root: true, exclude: :avatar)
     end
   end
   get '/users/:user_id' do
     requires_role! :admin
     json :user => @user
   end
+  get '/users/:user_id/avatar' do
+    content_type 'image/png' # probably wrong, but it works
```

Comment:
I was avoiding a schema change. I think I'll fix this when I add the user's actual name.